client/lib/user-agent/index.js

```diff
@@ -17,5 +17,4 @@ import UserAgent from 'express-useragent';
 
 	Note: we also import this lib server-side in server/boot/index.js
  */
-
-export default UserAgent.parse( navigator.userAgent );
+export default UserAgent.parse( typeof window !== 'undefined' ? window.navigator.userAgent : '' );
```

Comment:
Maybe add the type check before parsing? Why parse `''`?
```suggestion
export default typeof window !== 'undefined' ?  UserAgent.parse( window.navigator.userAgent ) : '';
```

Reply:
> Why parse ''? 

Since `UserAgent.parse` returns an object. This way I would have a consistent return.

Reply:
Good point. I'd still go for this.

```js
export default typeof window !== 'undefined' ? UserAgent.parse( window.navigator.userAgent ) : {};
```